Set the "no metricbeat data" alert throttle period correctly

Crazybus opened the pull request:
This used to be defaulting to what was in template.py instead of
correctly being updated to the `throttle` value in `kuberwatcher.yml`

Crazybus committed:
Set the "no metricbeat data" alert throttle period correctly

This used to be defaulting to what was in template.py instead of
correctly being updated to the `throttle` value in `kuberwatcher.yml`

fatmcgav commented on kuberwatcher.py:
```diff
@@ -245,7 +245,7 @@ def main(es, defaults):
     watches = generate_watch(pods)
-    watches['metricbeat'] = add_alerts(metricbeat_template, defaults['alerts'], defaults.get('reply_to', None))
+    watches['metricbeat'] = add_alerts(metricbeat_template, defaults['alerts'], defaults.get('throttle', 3600000), defaults.get('reply_to', None))
     watches['metricbeat']['metadata']['message'] = 'No metricbeat data has been recieved in the last 5 minutes! <{0}|kibana>'.format(defaults['kibana_url'])
```
`last 5 minutes!` is no longer accurate? 

https://humanfriendly.readthedocs.io/en/latest/api.html#humanfriendly.format_timespan Could be quite cool, if pulling in another lib isn't an issue :)

fatmcgav replied:
Ah, ignore that comment... Confusing `throttle` with `window`... :doh:

fatmcgav replied:
Actually, I think there's still something there... 

Does overriding the `watcher.window` item actually change this `message`?

Crazybus replied:
> Does overriding the watcher.window item actually change this message?

 Nope, but I'm working on another PR to make sure that the window itself is actually configurable for the "no metricbeat data" alert and that the text is also updated properly ( including other useful optional metadata like cluster name).

fatmcgav replied:
Cool, ok :)

fatmcgav reviewed (changes requested):
One comment around the `message` value. 

Otherwise, changes all look good :)

fatmcgav commented:
Also, does this need some new tests to explicitly override the `throttle` config item?

Crazybus commented:
> Also, does this need some new tests to explicitly override the throttle config item?

Great idea, adding an extra couple of tests now.

fatmcgav commented:
> > Also, does this need some new tests to explicitly override the throttle config item?
> 
> Great idea, adding an extra couple of tests now.

Cool, ok... I'll hold off approving just yet then...

Crazybus committed:
Add tests to make sure that throttle is properly overridden

The adding of alerts used to be handled by render_template but it was
pulled out at some point and the tests were not moved with it.

fatmcgav reviewed (approved):
LGTM
